recipes/sklearn-contrib-lightning/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{%set name = "sklearn-contrib-lightning" %}
+{%set version = "0.2.0" %}
+{%set hash_type = "md5" %}
+{%set hash_val = "cf373cf22e9c1b0c11133589d3b14926" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
You will need to use `numpy x.x` because (I believe) the package uses numpy ABI. That will "bake" the numpy version in the build string and ensure the proper numpy version will always be installed.